src/services/TestResultsService.ts

```diff
@@ -429,18 +431,26 @@ export class TestResultsService {
       });
   }
 
-  public isMissingRequiredCertificateNumberOnAdr(payload: ITestResultPayload): boolean {
+  private isMissingRequiredCertificateNumber(typeFunc: (arg1: any) => boolean, payload: ITestResultPayload) {
```

Comment:
I like the solution! Can you just rename `arg1` to `testType` ?
Also, don't we have an interface for the testType?

Reply:
Thank you!

Reply:
revised